components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/RDBMSSessionDataStore.java

```diff
@@ -183,81 +151,17 @@
-        } else {
-            sqlInsertDELETE = SQL_INSERT_DELETE_OPERATION;
-        }
-        if (!StringUtils.isBlank(deleteSTORETaskSQL)) {
-            sqlDeleteSTORETask = deleteSTORETaskSQL;
-        }
-        if (!StringUtils.isBlank(deleteTempSTORETaskSQL)) {
-            sqlDeleteTempDataTask = deleteTempSTORETaskSQL;
-        } else {
-            sqlDeleteTempDataTask = SQL_DELETE_TEMP_STORE_OPERATIONS_TASK;
-        }
-
-        if (!StringUtils.isBlank(deleteDELETETaskSQL)) {
-            sqlDeleteDELETETask = deleteDELETETaskSQL;
-        } else {
-            sqlDeleteDELETETask = SQL_DELETE_DELETE_OPERATIONS_TASK;
-        }
-        if (!StringUtils.isBlank(selectSQL)) {
-            sqlSelect = selectSQL;
-        }
-
-        String deleteChunkSizeString = IdentityUtil.getProperty("JDBCPersistenceManager.SessionDataPersist" +
-                ".SessionDataCleanUp.DeleteChunkSize");
-        if (StringUtils.isNotBlank(deleteChunkSizeString)) {
-            deleteChunkSize = Integer.parseInt(deleteChunkSizeString);
-        }
-
-        if (StringUtils.isNotBlank(deleteExpiredDataTaskSQL)) {
-            sqlDeleteExpiredDataTask = String.format(deleteExpiredDataTaskSQL, deleteChunkSize);
-        }
+        enablePersist = getBooleanProperty(FrameworkConstants.SessionDataStoreConstants.PERSIST_ENABLE,
+                FrameworkConstants.SessionDataStoreConstants.DEFAULT_ENABLE_PERSIST);
+        deleteChunkSize = getIntProperty(FrameworkConstants.SessionDataStoreConstants.GET_DELETE_CHUNK_SIZE,
+                FrameworkConstants.SessionDataStoreConstants.DEFAULT_DETELE_CHUNK_SIZE);
 
         if (!enablePersist) {
             log.info("Session Data Persistence of Authentication framework is not enabled.");
         }
-        String isCleanUpEnabledVal = IdentityUtil.getProperty("JDBCPersistenceManager.SessionDataPersist.SessionDataCleanUp" +
-                ".Enable");
-
-        String isOperationCleanUpEnabledVal = IdentityUtil.getProperty("JDBCPersistenceManager.SessionDataPersist" +
-                ".OperationDataCleanUp.Enable");
-
-        if (StringUtils.isNotBlank(isCleanUpEnabledVal)) {
-            sessionDataCleanupEnabled = Boolean.parseBoolean(isCleanUpEnabledVal);
-        }
-        if (StringUtils.isNotBlank(isOperationCleanUpEnabledVal)) {
-            operationDataCleanupEnabled = Boolean.parseBoolean(isOperationCleanUpEnabledVal);
-        }
-
-        if (sessionDataCleanupEnabled || operationDataCleanupEnabled || tempDataCleanupEnabled) {
+        sessionDataCleanupEnabled = getBooleanProperty(FrameworkConstants.SessionDataStoreConstants.SESSION_DATA_CLEANUP_ENABLE,
+                FrameworkConstants.SessionDataStoreConstants.DEFAULT_SESSION_DATA_CLEANUP_ENABLED);
+        if (sessionDataCleanupEnabled || tempDataCleanupEnabled) {
             long sessionCleanupPeriod = IdentityUtil.getCleanUpPeriod(
                     CarbonContext.getThreadLocalCarbonContext().getTenantDomain());
```

Comment:
```suggestion
            long sessionCleanupPeriod = IdentityUtil.getCleanUpPeriod(CarbonContext.getThreadLocalCarbonContext().getTenantDomain());
```
wdyt? does it exceed the line length?